parsers/ua-parser-php-3/composer.json

```diff
@@ -2,6 +2,9 @@
     "require": {
         "ua-parser/uap-php": "^3.4.6",
         "thadafinser/package-info": "^1.0",
-        "mimmi20/json-class": "~2.0.1"
+        "symfony/yaml": "^4.2",
+        "symfony/filesystem": "^4.2",
+        "symfony/finder": "^4.2",
+        "symfony/console": "^4.2"
```

Comment:
Why was it necessary to add these dependencies?

Reply:
The UA Parser removed these direct dependencies and added them as succests.

Reply:
see ua-parser/uap-php#45